client-app/src/components/AboutComponent.js

```diff
@@ -14,6 +14,60 @@ const AboutComponent = () => {
           downloading the extracted data as a Word document, ensuring seamless accessibility and
           usability.
         </p>
+
+        <h3>Example Inputs and Outputs</h3>
+        <ul className="examples-list">
+        <li>
+            <strong>Input 1:</strong> 
+            <ul>
+              <li><strong>Uploaded File:</strong> ORCA log file (`.txt`)</li>
+              <li><strong>Search Terms:</strong> `CARTESIAN COORDINATES`</li>
+              <li><strong>Line Specification:</strong> `FIRST`</li>
+              <li><strong>Sections:</strong> `1`</li>
+            </ul>
+            <strong>Output:</strong> Extracted data in `output.docx` containing the first occurrence of `CARTESIAN COORDINATES` in sections 1-5.
+          </li>
+          <li>
+            <strong>Input 2:</strong> 
+            <ul>
+              <li><strong>Uploaded File:</strong> ORCA log file with molecular energy data</li>
+              <li><strong>Search Terms:</strong> `TOTAL ENERGY, FINAL GRADIENT`</li>
+              <li><strong>Line Specification:</strong> `WHOLE`</li>
+              <li><strong>Sections:</strong> `2`</li>
+            </ul>
+            <strong>Output:</strong> Extracted data in `output.docx` containing all lines with `TOTAL ENERGY` and `FINAL GRADIENT` from sections 2 and 4.
+          </li>
+        </ul>
+        
+        <h3>Step-by-Step Guide</h3>
+        <ol className="how-to-guide">
+          <li>Upload a `.txt` file containing ORCA log data using the file upload option.</li>
+          <li>Enter keywords in the search field (e.g., `CARTESIAN COORDINATES`). Press **Enter** or use commas to add multiple terms.</li>
+          <li>Select how to specify lines (e.g., `FIRST`, `LAST`, `WHOLE`) and provide additional details like line numbers if required.</li>
+          <li>Define sections to search within (e.g., `1-2` or `1,2`).</li>
+          <li>Submit the search query to review and validate your criteria.</li>
+          <li>Preview the extracted data using the **Preview** button.</li>
+          <li>Download the results as a Word document by clicking the **Download Output** button.</li>
+        </ol>
+
+        <h3>Key Features</h3>
+        <ul>
+          <li><strong>File Upload:</strong> Accepts `.txt` files for ORCA data processing.</li>
+          <li><strong>Dynamic Search:</strong> Allows multiple search terms and criteria for precise data extraction.</li>
```

Comment:
We have not yet implemented the functionality to allow multiple search terms. Could you please update it accordingly?